3-meetings.md

```diff
@@ -21,9 +21,13 @@ Things you can discuss on a meeting:
 
 Any member can schedule a meeting.
 
-1. Create a pad for the meeting topics and the meeting notes.
-2. Pick a date (preferably not during the social evening).
-3. Put the meeting in the calendar with a link to the meeting notes, announce it in the main channel and put a link in the changelog.
+1. Create a pad for the meeting topics and the meeting notes. Use the pad of the previous meeting to see what it should look like.
+1. Pick a date (preferably not during the social evening).
+1. Announce the meeting in the Main channel and in the Changelog **at least a week in advance**. The announcement should include:
```

Comment:
i'm still of the opinion the Changelog is only for logging changes **after** they happen
but recognize that the Main channel often is too noisy to ensure the most amount of people see the announcement
would setting the requirement of using an `@all` tag in the Main channel be enough to not need to use Changelog for this?

Reply:
I don't think `@all` in the main channel is good enough. Some of my thoughts on this:

* When you get a ping, it's hard to see what you got the ping for. People will have to search through the main channel to see what they got the ping from.
* A ping is basically "either check it out now, or forget about it". The changelog is much easier to simply go through it on your own time.

I agree there is a slight difference between this guideline and what a changelog is conceptually, but I think we need to be pragmatic about the purpose and look at it from a viewpoint of "what is the effect of posting this in changelog" instead of "what does conceptually fit best in a changelog".